Source/CoreLibrary/Controls/MultiSelectCombobox/BasicStructure.cs

```diff
@@ -51,6 +51,10 @@ private RichTextBox RichTextBoxElement
                     richTextBoxElement.RemoveHandler(CommandManager.PreviewExecutedEvent, new ExecutedRoutedEventHandler(SetClipboardTextWithCommandCancelled));
                     richTextBoxElement.DragEnter -= OnDragEnter;
                     richTextBoxElement.Drop -= OnDragDrop;
+                    foreach (var textblock in richTextBoxElement.GetParagraph().Inlines.Select(i => i.GetTextBlock()).Where(i => i != null))
```

Comment:
Can you please use - null conditional operator?
i.e. - foreach (var textblock in richTextBoxElement.GetParagraph()?.Inlines?.Select(i => i.GetTextBlock())?.Where(i => i != null))

Reply:
Updated.

Reply:
Updated.